integration/tests/neon_evm/solana_native/test_scheduled_transactions.py

```diff
@@ -116,6 +116,7 @@ def test_scheduled_trx_wrong_index(
         with pytest.raises(solana.rpc.core.RPCException, match=InstructionAsserts.TRANSACTION_TREE_INVALID_DATA):
             evm_loader.create_tree_account(neon_user, treasury_pool, tx.encode(), environment.sol_mint_id)
 
+    @pytest.mark.eip_1559
```

Comment:
please remove eip_1559 marks from scheduled transactions tests
there are different type of transactions